### PR TITLE
Update docs from Darga to Euwe

### PR DESCRIPTION
**Pull request by cybette**
Updating the getting started guides for the Euwe release

@sergio-ocon please review, let me know if there are files I missed

@geertj are you doing the GCE builds? perhaps you can instruct me how to do it and I can take care of them going forward

@hayesr this should be merged when Euwe GA goes live

@cybette note to self: update GCE screenshots?

### CODE REVIEW

File: site/docs/get-started/docker.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ is available).
 Pull the ManageIQ docker image:
 
 ```bash
-$ sudo docker pull manageiq/manageiq:latest-darga
+$ sudo docker pull manageiq/manageiq:latest-euwe
```
Comment (simaishi) [inline site/docs/get-started/docker.md]: `latest-darga` and `latest-euwe` are nightly builds. A release tag (e.g. `euwe-1`) should be used instead?

Comment (cybette) [inline site/docs/get-started/docker.md]: good point, I'll change the tag. thanks @simaishi !

btw I see you've also uploaded euwe-1-rc2 GCE build to Google Storage. I assume you'll be taking care of future releases as well. thanks!

**Comment by sergio-ocon** [comment]
Some screenshots are from Euwe (the service catlog), the rest has not changes much, so we can update it when testing it

File: site/docs/get-started/vagrant.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Execute the following commands to create a new Vagrant box:
 ```bash
 $ mkdir manageiq
 $ cd manageiq
-$ vagrant init manageiq/darga
+$ vagrant init manageiq/euwe
```
Comment (sergio-ocon) [inline site/docs/get-started/vagrant.md]: Wouldn't be better to use {{release.tag}} here as you did for docker?

File: site/docs/get-started/docker.md [hunk 3]
```diff
@@ -8,18 +8,20 @@ ManageIQ project makes available on the [Docker Hub](https://hub.docker.com/r/ma
...
 is available).
 
+{% assign release = site.data.releases["stable"] %}
+
 ### Step 1: Download and deploy the appliance
 
 Pull the ManageIQ docker image:
 
 ```bash
-$ sudo docker pull manageiq/manageiq:latest-darga
```
Comment (cybette) [inline site/docs/get-started/docker.md]: The "latest" tag will be valid, but I think it's better to use a stable image for the getting started guide. If the user is more experienced they can find out easily the other options to use, including latest.

File: site/docs/get-started/vagrant.md (hunk 2 above)
Comment (sergio-ocon) [inline site/docs/get-started/vagrant.md]: Thanks @cybette

**Pull request merged**